packages/script/src/script-invocation-scope.ts

```diff
@@ -49,24 +43,4 @@ export class ScriptInvocationScope<
       () => [] as unknown as TReturn
     );
   }
-
-  /**
-   * Submits a script transaction to the blockchain.
-   */
-  async call<T = TReturn>(): Promise<FunctionInvocationResult<T>> {
-    assert(this.program.account, 'Provider is required!');
-
-    const transactionRequest = await this.getTransactionRequest();
-    const { maxFee } = await this.getTransactionCost();
-    await this.fundWithRequiredCoins(maxFee);
-
-    const response = await this.program.account.sendTransaction(transactionRequest);
-
-    return FunctionInvocationResult.build<T>(
-      this as unknown as InvocationScopeLike,
-      response,
-      false,
-      this.program
-    );
-  }
```

Comment:
This is the implementation of `BaseInvocationScope.call`:
```ts
  async call<T = TReturn>(): Promise<FunctionInvocationResult<T>> {
    assert(this.program.account, 'Wallet is required!');

    const transactionRequest = await this.getTransactionRequest();

    const { maxFee, gasUsed } = await this.getTransactionCost();

    if (gasUsed.gt(bn(transactionRequest.gasLimit))) {
      throw new FuelError(
        ErrorCode.GAS_LIMIT_TOO_LOW,
        `Gas limit '${transactionRequest.gasLimit}' is lower than the required: '${gasUsed}'.`
      );
    }

    await this.fundWithRequiredCoins(maxFee);

    const response = await this.program.account.sendTransaction(transactionRequest);

    return FunctionInvocationResult.build<T>(
      this.functionInvocationScopes,
      response,
      this.isMultiCall,
      this.program as AbstractContract
    );
  }
```

Reply:
the `this.isMultiCall` near the end is `false` for a `ScriptInvocationScope` because its base class is `FunctionInvocationScope` which sets the `isMultiCall` property to `false` in its constructor.